Sources/TuistAutomation/Utilities/TargetBuilder.swift

```diff
@@ -49,6 +52,8 @@ public enum TargetBuilderError: FatalError {
             return "The scheme \(scheme) cannot be built because it contains no buildable targets."
         case let .buildProductsNotFound(path):
             return "The expected build products at \(path) were not found."
+        case let .cantDeterminePlatform(target):
+            return "Only single platform targets supported. The target \(target.name) specifies multiple supported platforms (\(target.supportedPlatforms.map(\.rawValue).joined(separator: ", ")))."
```

Comment:
Would it be more accurate to name this case `.multiPlatformTargesNotSupported(target)`? Is this something that you think is sensible to support in the future?

Reply:
i was thinking in the future we could have a `--platform` flag passed in to support multi-platform targets, so this would cover the case where that was missing.  the error messaging could be updated to be more prescriptive about recovery steps then as well. 

Happy to rename for now if we want tho.